src/content/docs/en/guides/internationalization.mdx

```diff
@@ -152,6 +152,10 @@ Set this option when all routes will have their `/locale/` prefix in their URL a
 - URLs without a locale prefix, (e.g. `example.com/blog/`) will return a 404 (not found) status code.
 - The home URL (`/`) redirects to `/<defaultLocale>`.
 
+#### `redirectToDefaultLocale: true`
+
+When `prefixDefaultLocale` is `true`, the redirect form the home URL to `/<defaultLocale>` is disabled. 
```

Comment:
```suggestion
When `prefixDefaultLocale` is `true`, `redirectToDefaultLocale: false` will disable redirecting from the home URL to `/<defaultLocale>`. 
```

Some alternatives:
- When `prefixDefaultLocale` is `true`, `redirectToDefaultLocale` controls whether to redirect from the home URL to `/<defaultLocale>`. 
- When `prefixDefaultLocale` is `true` and `redirectToDefaultLocale` is `false`, redirecting from the home URL to `/<defaultLocale>` is disabled.

Reply:
I like the first alternative!